packages/nimble-components/src/dialog/template.ts

```diff
@@ -6,6 +6,7 @@ export const template = html<Dialog>`
         <dialog
             ${ref('dialogElement')}
             role="dialog"
+            part="dialog"
```

Comment:
Let's name it `control` that's how [FAST named the part of their dialog with the actual role dialog](https://github.com/microsoft/fast/blob/507d2fef836f75b072d0c05c9e37b7c59959ff51/packages/web-components/fast-foundation/src/dialog/dialog.template.ts#L30) so we can align with that.